CHANGELOG

```diff
@@ -1,3 +1,7 @@
+Version 0.3.1 (2019-09-26)
+--------------------------
+Update referers.yml to latest version (#75)
```

Comment:
There should be one entry for each commit; and that should correspond to one issue in the issue tracker.